src/hooks/screen/useCreateChat.ts

```diff
@@ -124,6 +125,21 @@ const useCreateChat = () => {
     }
   };
 
+  const createAnonymousChat = async (selectedUser) => {
+    try {
+      setLoadingCreateChat(true);
+      const response = await getOrCreateAnonymousChannel(selectedUser?.user?.userId);
+      const chatData = createChannelJson(response, selectedUser);
+      const channelList = ChannelList.fromMessageAnonymouslyAPI(chatData);
+      await channelList.saveIfLatest(localDb);
+      handleMemberSchema(response);
+      goToChatScreen(channelList, GROUP_INFO);
+    } catch (e) {
+      setLoadingCreateChat(false);
+      console.log({e}, 'error create chat');
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling in the `createAnonymousChat` function could be improved. Currently, if an error occurs, the loading state is set to false and the error is logged. However, there's no feedback to the user about the failure. Consider adding a user-friendly error message.

```diff
  catch (e) {
-   setLoadingCreateChat(false);
-   console.log({e}, 'error create chat');
+   try {
+     setLoadingCreateChat(false);
+     SimpleToast.show('Failed to create anonymous chat. Please try again.');
+     console.error({e}, 'error create chat');
+   } catch (toastError) {
+     console.error('Failed to show toast', toastError);
+   }
  }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->